lib/ddtrace/contrib/integration.rb

```diff
@@ -46,9 +46,9 @@ module Contrib
     #
     #   module Instrumentation
     #     def api_request!(env)
-    #       Datadog.tracer.trace('billing.request',
+    #       Datadog::Tracing.trace('billing.request',
     #                            type: 'http',
-    #                            service: Datadog.configuration[:billing_api][:service]) do |span|
+    #                            service: Datadog::Tracing.configuration[:billing_api][:service]) do |span|
```

Comment:
Is this still correct? E.g. aren't we moving away from integration-specific service naming?

(Same note applies to a few other places in this file)

Reply:
Integration-specific service naming still applies in the case that the span measures an external service.

The `service_name` becomes both the `service` field, and the `peer.service_name` tag, basically. Eventually it will be only the `peer.service_name` tag when the backend supports it.